prime-router/src/test/kotlin/fhirengine/engine/CustomFhirPathFunctionTest.kt

```diff
@@ -140,5 +140,6 @@ class CustomFhirPathFunctionTest {
         assertThat(
             (result[0] as StringType).value
         ).isEqualTo(loincCode)
+        unmockkObject(LivdLookup, Metadata)
```

Comment:
Tip: using @AfterEach is generally cleaner than cleaning up directly in your test

Reply:
I didn't think it was normal to put an @AfterEach when only half your tests use a thing.

Reply:
I assume it's a noop when the object isn't currently mocked. It looks like there is also the option to call `unmockkAll` in the `@AfterEach` block to really clean everything up. If some tests are using the unmocked versions of those objects then the current way is probably the right way to do it.

https://mockk.io/#object-mocks

Reply:
Some combination of `unmockkAll` and `clearAllMocks` seems to be used in some combination of `@BeforeEach` and `@AfterEach` in some test files, but it's all inconsistent. I hope we can discuss this next eng sync and establish a best practice for this moving forward. There may be some cases where we don't want to unmock/clear everything between tests, so just need to talk through the options, and then document what we decide so we can reference that when making these kinds of decisions and hopefully get to some consistency eventually. Either way, it seems like we're on the same page that this is fine for this kind of quick fix, so all good for now 👍